google-cloud-spanner-hibernate-samples/quarkus-jpa-sample/pom.xml

```diff
@@ -13,22 +13,25 @@
   <artifactId>quarkus-jpa-sample</artifactId>
 
   <properties>
-    <surefire-plugin.version>2.22.0</surefire-plugin.version>
-    <maven.compiler.target>1.8</maven.compiler.target>
+    <compiler-plugin.version>3.8.1</compiler-plugin.version>
+    <maven.compiler.parameters>true</maven.compiler.parameters>
     <maven.compiler.source>1.8</maven.compiler.source>
-    <quarkus.version>0.27.0</quarkus.version>
+    <maven.compiler.target>1.8</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <maven.compiler.parameters>true</maven.compiler.parameters>
-    <compiler-plugin.version>3.8.1</compiler-plugin.version>
+    <quarkus-plugin.version>1.0.1.Final</quarkus-plugin.version>
+    <quarkus.platform.artifact-id>quarkus-universe-bom</quarkus.platform.artifact-id>
+    <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
```

Comment:
It's a bit odd to use properties for group id and artifact id, but OK.

Reply:
yeah i wasn't sure the reason but it was generated by quarkus.